.github/workflows/new-prs-labeler.yml

```diff
@@ -0,0 +1,276 @@
+backend:AArch64:
+  - llvm/include/llvm/IR/IntrinsicsAArch64.td
+  - llvm/lib/Target/AArch64/**/*
+  - llvm/test/**/*AArch64/**/*
+  - clang/lib/Basic/Targets/AArch64*
+  - clang/lib/Driver/ToolChains/Arch/AArch64*
+  - clang/lib/CodeGen/Targets/AArch64.cpp
+  - clang/include/clang/Basic/BuiltinsAArch64*
+
+backend:ARM:
+  - llvm/include/llvm/IR/IntrinsicsARM.td
+  - llvm/lib/Target/ARM/**/*
+  - llvm/test/**/*ARM/**/*
+  - clang/lib/Basic/Targets/ARM*
+  - clang/lib/Driver/ToolChains/Arch/ARM*
+  - clang/lib/CodeGen/Targets/ARM.cpp
+  - clang/include/clang/Basic/BuiltinsARM*
+
+backend:DirectX:
+  - llvm/include/llvm/IR/IntrinsicsDirectX.td
+  - llvm/**/Target/DirectX/**/*
+  - llvm/**/dxil-dis/**/*
+  - clang/lib/Basic/Targets/DirectX*
+
+backend:MSP430:
+  - llvm/include/llvm/IR/IntrinsicsMSP430.td
+  - llvm/lib/Target/MSP430/**/*
+  - llvm/test/**/MSP430/**/*
+  - clang/lib/Basic/Targets/MSP430*
+  - clang/lib/Driver/ToolChains/Arch/MSP430*
+  - clang/lib/CodeGen/Targets/MSP430.cpp
+  - clang/include/clang/Basic/BuiltinsMSP430*
+
+backend:RISC-V:
+  - llvm/**/*RISCV*/**/*
+  - llvm/**/*risv*/**/*
+  - clang/**/*RISCV*/**/*
+  - clang/**/*risv*/**/*
+
+backend:Sparc:
+  - llvm/include/llvm/IR/IntrinsicsSparc.td
+  - llvm/lib/Target/Sparc/**/*
+  - llvm/test/**/Sparc/**/*
+  - clang/lib/Basic/Targets/Sparc*
+  - clang/lib/Driver/ToolChains/Arch/Sparc*
+  - clang/lib/CodeGen/Targets/Sparc.cpp
+  - clang/include/clang/Basic/BuiltinsSparc*
+
+backend:X86:
+  - llvm/include/llvm/IR/IntrinsicsX86.td
+  - llvm/include/llvm/Support/X86DisassemblerDecoderCommon.h
+  - llvm/include/llvm/TargetParser/X86*
+  - llvm/lib/TargetParser/X86*
+  - llvm/**/**/X86/**/*
+  - llvm/utils/TableGen/X86*
+  - clang/lib/Basic/Targets/X86/**/*
+  - clang/lib/Driver/ToolChains/Arch/X86.*
+  - clang/lib/CodeGen/Targets/X86.*
+  - clang/test/CodeGen/X86/**/*
+  - clang/include/clang/Basic/BuiltinsX86*
+
+backend:loongarch:
+  - llvm/include/llvm/IR/IntrinsicsLoongArch.td
+  - llvm/lib/Target/LoongArch/**/*
+  - llvm/test/**/LoongArch/**/*
+  - clang/lib/Basic/Targets/LoongArch*
+  - clang/lib/Driver/ToolChains/Arch/LoongArch*
+  - clang/lib/CodeGen/Targets/LoongArch.cpp
+  - clang/include/clang/Basic/BuiltinsLoongArch*
+
+clang:
+  - all: ['clang/**/*', '!clang/clang-tools-extra/**/*']
```

Comment:
I think this will lead to duplication between `clang` and `clang:*` labels for components.
We should either list the rest of directories under `/clang/` or just don't apply it at all.